### ISSUE TITLE
Wishlist support for multiple urls

### ISSUE THREAD

**Issue by legion0** [DestinyItemManager/DIM#4917]
https://www.d2checklist.com/gear has support for separate PVE/PVP lists and in each of those it supports multiple urls separated by "|".

We should
a. support multiple urls (with some separator, e.g. "|"), this can be done without needing to distinguish between them and only requires fetching and concatenation (with some minor tweaks to drop the titles maybe).

b. consider separate PVP/PVE lists, probably warrants a separate issue and is not a blocker for implementing a, it only makes sense if we indicate this in the UI somehow.

**Comment by bhollis** [comment]
a. Just make a new wishlist that combines the different files.
b. Submit a PR to @48klocs' wishlist repo with different lists.

**Comment by legion0** [comment]
That is a very laborious way of doing this, what happens when the original wish list i added to changes? (e.g. one from 48klocs repo) I will have to manually track that and make sure to update my combined wish list and re-push it.
Having multiple url support means that i can put the original list and my short custom list as 2 urls and don't need to worry about the original changing.

**Comment by legion0** [comment]
Hi, please reopen, this is not working as of today, tried the following and got an error about invalid url

https://raw.githubusercontent.com/48klocs/dim-wish-list-sources/master/voltron.txt|https://raw.githubusercontent.com/48klocs/dim-wish-list-sources/master/choosy_voltron.txt

**Comment by kyleshay** [comment]
ah - this feature is currently only on the beta. we push to the production version on Sunday.

**Comment by legion0** [comment]
I checked beta too, forgot to mention, same error
P.s. generally good practice to close issues once they land, i.e. reached prod

**Comment by kyleshay** [comment]
Try clearing your wishlist and then pasting the link back in. 

It's working for me on beta, I'll need some more information. Console logs, what the error says, etc.

**Comment by legion0** [comment]
Thanks, that seemed to work.

p.s. you might want to open a separate issue for the display, next to the clear button it only shows the title and description of one of the lists only, its not that bad but worth while having its own issue to track how many people complain about it.

**Comment by kyleshay** [comment]
Ah yeah that's part of the longer term plan. In the interm the first name and description that is matched from the first URL will be used.

**Comment by delphiactual** [comment]
Issues are and will always be closed when they are fixed in beta. Even, if you could point us to some "best practices" that state only to close issues when they reach prod that still would not change this fact; as this repo is based on beta, not prod.

**Comment by legion0** [comment]
@delphiactual you do you boo, just saying that for user facing issues it makes sense to close them when the user can actually verify that they are fixed in prod and not in a testing environment, since you can't reasonably expect your "normal" user to use the beta version, nor should you.
I don't understand what you mean by "as this repo is based on beta", repos are not based on release environments, release environments are releases of the repo, not the other way around.
But we are going off topic and since we don't work together it's not my place to tell you how to do your job right, was just trying to help, gonna stop now.

**Comment by kyleshay** [comment]
normal users aren't checking github, they check the release notes in the app, if anything. of which, this change isn't listed there yet.
